components/admin_real/desktop/DeleteReservationButton.jsx

```diff
@@ -3,16 +3,31 @@ import { Delete } from "@mui/icons-material";
 import { Button } from "@mui/material";
 import Modal from "antd/lib/modal/Modal";
 import { deleteReservation } from "../shared/reservationTableUtils";
+import { openErrorNotification } from "../shared/reservationTableUtils";
+import { openSuccessNotification } from "../shared/reservationTableUtils";
+import { TableContext } from "../../states/ReservationTableContextProvider";
+
 const DeleteReservationButton = ({ reservationId }) => {
+  const context = React.useContext(TableContext);
   const [modalOpen, setModalOpen] = useState(false);
+  const handleSuccess = () => {
+    context.getReservations();
+    openSuccessNotification("success");
+  };
   return (
     <div>
       <Modal
         title={"Are you sure you want to delete this reservation?"}
         style={{}}
         visible={modalOpen}
         onOk={() => {
-          deleteReservation(reservationId);
+          deleteReservation(reservationId).then((results) =>
+            results.error
```

Comment:
are you sure this can happen? I think the error would not be set here but on a `catch` of the promise...

Reply:
I am sure